visionmetrics/prediction_filters.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
```

Comment:
I guess, going forward, we use torch.tensor for everything. replace numpy with torch.